src/app_engine/bigquery/enums.json

```diff
@@ -18,5 +19,13 @@
       "ROOM_ID": "room_id",
       "EVENT_TIME_MS": "event_time_ms"
     }
+  },
+
+  "ClientType": {
+    "UNKNOWN": 0,
+    "JS": 1,
+    "DESKTOP": 2,
```

Comment:
What's desktop?
Do we need to differentiate Chrome vs Chrome App?
What about linux/mac/win?